doc/classes/AnimationNode.xml

```diff
@@ -43,12 +43,6 @@
 				When inheriting from [AnimationRootNode], implement this virtual method to return a list of the properties on this animation node. Parameters are custom local memory used for your animation nodes, given a resource can be reused in multiple trees. Format is similar to [method Object.get_property_list].
 			</description>
 		</method>
-		<method name="_has_filter" qualifiers="virtual const">
-			<return type="bool" />
-			<description>
-				When inheriting from [AnimationRootNode], implement this virtual method to return whether the blend tree editor should display filter editing on this animation node.
-			</description>
-		</method>
```

Comment:
```
Error: Validate extension JSON: API was removed: classes/AnimationNode/methods/_has_filter
```

It is not permitted to remove public APIs, such as `AnimationNode.is_path_filtered(path:NodePath)`, `AnimationNode.set_filter_path(path:NodePath,enable:bool)`, `AnimationNode.set_filter_enabled(enable:bool)`,  `AnimationNode.is_filter_enabled()` or the virtual function `AnimationNode._has_filter()`. The property `filter_enabled` may also be kept for script compatibility, but it is not required to do so for GDExtension support.

These `AnimationNode` methods must be kept but marked deprecated, such as using `is_deprecated="true"` in this documentation XML. I would suggest that it would be good practice to implement them if possible, for example accessing the attached `AnimationTrackFilter` if possible, or returning empty if this is not possible.